launching-software/infrastructure.md

```diff
@@ -145,11 +143,11 @@ Thus in order to keep our rates low, it's extremely important to bill
 infrastructure costs, including non-production costs, to agency partners
 wherever possible. If the work you are doing is in support of a project which
 has an inter-agency agreement (IAA), you _must_ register your system with
-[##infrastructure](https://gsa-tts.slack.com/messages/C039MHHF8/), including the
+[#infrastructure](https://gsa-tts.slack.com/messages/C039MHHF8/), including the
```

Comment:
Since this link is used in at least 3 places, maybe you want to use a [reference link](https://riptutorial.com/markdown/example/2215/reference-link) instead?